### PR TITLE
Add SAML metadata example file

### PR DESCRIPTION
**Pull request by german-b**
Copied from https://gist.github.com/p2tris/b6919b18b921a880de93badefb946eb6

Needed to update the SAML metadata example file in the support article: https://support.glia.com/technical-requirements/saml-overview

### CODE REVIEW

File: SAML/example_metadata.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://client_name.app.salemove.com">
```
Comment (p2tris) [inline SAML/example_metadata.xml:1]: I think two examples would be good, one for .com and one for .eu

Comment (kadriiiii) [reply SAML/example_metadata.xml:1]: Do we still use "salemove.com" in this context or is it possible to use "glia.com" as well?

Comment (german-b) [reply SAML/example_metadata.xml:1]: Currently only salemove.com is used: https://salemove.atlassian.net/browse/TM-10809

Comment (kadriiiii) [reply SAML/example_metadata.xml:1]: Ok, good!

**Comment by german-b** [comment]
!squash

**Comment by german-b** [comment]
!merge

**Pull request merged**